forum/communications/serializers.py

```diff
@@ -1,10 +1,72 @@
 from rest_framework import serializers
+from .mongo_models import Room
+from .helpers import is_namespace_info_correct
+from bson.objectid import ObjectId
+from bson.errors import InvalidId
+
+
+class NamespaceInfoSerializer(serializers.Serializer):
+    user_id = serializers.IntegerField(required=True)
+    namespace = serializers.ChoiceField(choices=[
+        ("startup","startup"),
+        ("investor","investor")
+    ], required=True)
+    namespace_id = serializers.CharField(required=True)
+
+
+class RoomSerializer(serializers.Serializer):
+    participants = serializers.ListField()
+
+    def validate(self, data):
+        participants = data.get("participants")
+
+        if len(participants) != 2:
+            raise serializers.ValidationError("Only two participants in one room.")
+
+        for participant in participants:
+            if not NamespaceInfoSerializer(data=participant).is_valid():
+                raise serializers.ValidationError("Invalid participant.")
+            
+            namespace_obj = is_namespace_info_correct(participant)
+            
+            if not namespace_obj:
+                raise serializers.ValidationError("Incorrect participants data.")
+        
+        namespaces = [p.get("namespace") for p in participants]
+
+        if ("startup" not in namespaces) or ("investor" not in namespaces):
+            raise serializers.ValidationError("Room can be created only for investor and startup.")
+
+        return data
 
 
 class ChatMessageSerializer(serializers.Serializer):
-    # TODO: implement this serializer in an appropriate task
-    ...
+    room = serializers.CharField(required=True)
+    author = serializers.JSONField(required=True)
+    content = serializers.CharField(required=True)
+
+    def validate(self, data):
+        room_id = data.get("room")
+        author = data.get("author")
+
+        try:
+            room_id = ObjectId(room_id)
+        except InvalidId:
+            raise serializers.ValidationError("Invalid room id.")
+
+        room = Room.objects.filter(id=room_id).first()
```

Comment:
Change .first() to .exists() to simply check for existence, which is more efficient.